dev_guide/deployments.adoc

```diff
@@ -12,31 +12,46 @@ toc::[]
-template called a deployment configuration. Deployments are created manually
-or in response to triggered events.
+{product-title} deployments provide fine-grained management over applications
+based on a user-defined template called a deployment configuration.
+The deployment system in response to a deployment configuration will
+create a replication controller to run an application. Replication
+controllers are created in response to triggered events, or manually using:
 
-The deployment system provides:
+----
+$ oc deploy dc/foo --latest
+----
+
+[IMPORTANT]
+====
+Users should never manipulate replication controllers owned by
+deployment configurations. The deployment system makes sure changes to
+deployment configurations are propagated appropriately to replication
+controllers.
+====
 
-- A xref:creating-a-deployment-configuration[deployment configuration], which is a template for deployments.
+Features provided by the deployment system:
+
+- A xref:creating-a-deployment-configuration[deployment configuration], which is a template for running applications.
 - xref:triggers[Triggers] that drive automated deployments in response to events.
-- User-customizable xref:strategies[strategies] to transition from the previous deployment to the new deployment.
-- xref:rolling-back-a-deployment[Rollbacks] to a previous deployment.
-- Manual replication xref:scaling[scaling].
+- User-customizable xref:strategies[strategies] to transition from the previous version to the new version.
+- xref:rolling-back-a-deployment[Rollbacks] to a previous version either manually or automatically in case of deployment failure.
+- Manual replication xref:scaling[scaling] and autoscaling.
 
 The deployment configuration contains a version number that is incremented
-each time a new deployment is created from that configuration. In addition,
-the cause of the last deployment is added to the configuration.
+each time a new replication controller is created from that configuration.
+In addition, the cause of the last deployed replication controller is added
+to the deployment configuration.
 
 [[creating-a-deployment-configuration]]
 == Creating a Deployment Configuration
 
 A deployment configuration consists of the following key parts:
 
-- A replication controller template which describes the application to be deployed.
-- The default replica count for the deployment.
-- A deployment xref:strategies[strategy] which will be used to execute the deployment.
-- A set of xref:triggers[triggers] which cause deployments to be created automatically.
+- A pod template, which describes the application to be deployed.
+- The initial replica count for the replication controller.
+- A deployment xref:strategies[strategy], which will be used to deploy the application.
+- A set of xref:triggers[triggers] and xref:lifecycle-hooks[lifecycle hooks], which cause replication controllers to be created automatically.
```

Comment:
Move hooks into a separate bullet point.

*A set of hooks for executing custom behavior in different points during the lifecycle of a deployment.*